lms/views/oauth.py

```diff
@@ -2,11 +2,14 @@
 import jwt
 from pyramid.view import view_config
 from requests_oauthlib import OAuth2Session
+
 from lms.models.tokens import update_user_token, build_token_from_oauth_response
 from lms.models import find_user_from_state
 from lms.util import lti_params_for
 from lms.util.lti_launch import get_application_instance
 from lms.views.content_item_selection import content_item_form
+from lms.validation import parser
+from lms.validation import CANVAS_OAUTH_CALLBACK_ARGS
```

Comment:
Note how the names are imported from the `__init__.py` not from the submodules themselves